app/design/adminhtml/Magento/backend/Magento_Analytics/web/css/source/_module.less

```diff
@@ -109,8 +109,7 @@
 }
 
 .advanced-reports-subscription-close {
-  display: inline-block;
-  vertical-align: top;
+  display: block;
```

Comment:
Are you sure that we can change display inline-block to block? It can break UI.

Reply:
Due to float it is always inline-block.

See https://codepen.io/DanielRuf/pen/bjxgwm?editors=1100

Reply:
Due to float it is always handled as block.

See https://codepen.io/DanielRuf/pen/bjxgwm?editors=1100

Reply:
https://developer.mozilla.org/en-US/docs/Web/CSS/float

> As float implies the use of the block layout, it modifies the computed value of the display values, in some cases:

specified | computed
-- | --
inline | block
inline-block | block
inline-table | table
table-row | block
table-row-group | block
table-column | block
table-column-group | block
table-cell | block
table-caption | block
table-header-group | block
table-footer-group | block
flex | flex, but float has no effect on such elements
inline-flex | inline-flex, but float has no effect on such elements
other | unchanged